entrypoint.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/bash
-
-git config --global --add safe.directory /github/workspace
```

Comment:
I guess this pr depends on the "remove git" task then?

Reply:
It would, but I "fixed" it here: https://github.com/Vandebron/gh-mpyl/pull/33/files#diff-6040da3442be1c39607355675768292629b1dbebadd4fd299aca63e131ccab5eR112

Reply:
Turns out there's more usages of git that I had forgotten about (`ls-files`, etc). I might need put this one back into draft.